controllers/chat/group/changeChannelDetail.js

```diff
@@ -42,6 +42,7 @@ const changeChannelDetail = async (req, res) => {
         text: textDefaultUser,
         own_text: textOwnUser,
         other_text: textTargetUser,
+        better_type: 'change_channel_detail',
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of the `better_type` property with a hardcoded value 'change_channel_detail' seems to be an attempt to provide more context about the type of system message. However, it's not clear how this new property is going to be used in the rest of the codebase. If it's meant to replace the existing `type` property, then it might be better to change the value of `type` instead of introducing a new property. This would avoid potential confusion and maintain consistency across the codebase.

```diff
-        better_type: 'change_channel_detail',
+        type: 'change_channel_detail',
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->